sql/2021/structured-data/present_types.sql

```diff
@@ -0,0 +1,69 @@
+#standardSQL
+  # A count of pages which include each type of structured data
```

Comment:
This query seems like it'd benefit from a custom JS function to return a list of structured data types present on the page to avoid the need to extract each value individually from the JSON. (instead, one JSON extraction per page)

That should reduce a lot of the redundancy in the SELECT.


Not exactly the same but very similar to my feedback on [this Performance query](https://github.com/HTTPArchive/almanac.httparchive.org/pull/2316/files#r721871823). See the linked Accessibility query as an example of the pattern I'm suggesting.